ballerina-command/src/main/resources/cli-help/ballerina-update.help

```diff
@@ -1,15 +1,14 @@
 NAME
-    ballerina-update - Update to latest Ballerina command version
+       ballerina-update - Update to the latest Ballerina command version
 
 SYNOPSIS
-    ballerina update
+       ballerina update
+
 
 DESCRIPTION
-     Ballerina update updates to Ballerina command to latest version.
+       Update updates the Ballerina command to the latest version.
```

Comment:
Shall we say, `update command updates....`

Reply:
This was what we decided for the commands initially IINM (run, build, etc. still follows this format while there are some that have diverged).